components/data-oriented-programming/intro/Intro.js

```diff
@@ -38,7 +38,7 @@ export default function Intro() {
                                 </p>
 
                                 <p className={styles.introText3}>
-                                With its robust support, seamless integration with data constructs, and powerful features make Ballerina the top choice for efficient data handling and processing.
+                                Ballerina's robust support, seamless integration with data constructs, and powerful features make it the top choice for efficient data handling and processing.
```

Comment:
I prefer the previous case as it first mentions what it has and then brings out the name. Wdyt?

Reply:
The original sentence is grammatically incorrect. It should be either of the below.

`With its robust support, seamless integration with data constructs, and powerful features, Ballerina makes itself the top choice for efficient data handling and processing.`

`The robust support, seamless integration with data constructs, and powerful features make Ballerina the top choice for efficient data handling and processing.`

Reply:
Don't prefer `make itself` part.What if
```
With its robust support, seamless integration with data constructs, and powerful features, Ballerina stands out as the top choice for efficient data handling and processing.
```